tests/v2/validation/projects/projects_test.go

```diff
@@ -0,0 +1,301 @@
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a standard user and the user to the downstream cluster as cluster owner.")
+	standardUser, err := users.CreateUserWithRole(pr.client, users.UserConfig(), projects.StandardUser)
+	require.NoError(pr.T(), err, "Failed to create standard user")
+	standardUserClient, err := pr.client.AsUser(standardUser)
+	require.NoError(pr.T(), err)
+	err = users.AddClusterRoleToUser(pr.client, pr.cluster, standardUser, clusterOwner, nil)
+	require.NoError(pr.T(), err, "Failed to add the user as a cluster owner to the downstream cluster")
+
+	log.Info("Create a project (without any resource quota) and a namespace in the project.")
+	createdProject, createdNamespace, err := createProjectAndNamespace(standardUserClient, pr.cluster.ID, "", "", "", "", "", "")
+	require.NoError(pr.T(), err)
+
+	log.Info("Verify that the namespace has the label and annotation referencing the project.")
+	updatedNamespace, err := namespaces.GetNamespaceByName(standardUserClient, pr.cluster.ID, createdNamespace.Name)
+	require.NoError(pr.T(), err)
+	err = checkNamespaceLabelsAndAnnotations(pr.cluster.ID, createdProject.Name, updatedNamespace)
+	require.NoError(pr.T(), err)
+
+	log.Info("Verify that the namespace does not have the annotation: field.cattle.io/resourceQuota.")
+	err = checkAnnotationExistsInNamespace(standardUserClient, pr.cluster.ID, updatedNamespace.Name, resourceQuotaAnnotation, false)
+	require.NoError(pr.T(), err, "'field.cattle.io/resourceQuota' annotation should not exist")
+
+	log.Info("Create a deployment in the namespace with ten replicas.")
+	deployment, err := createDeployment(standardUserClient, pr.cluster.ID, updatedNamespace.Name, 10)
+	require.NoError(pr.T(), err, "Failed to create deployment in the namespace")
+
+	log.Info("Verify that there are ten pods created in the deployment and they are in Running state.")
+	err = charts.WatchAndWaitDeployments(standardUserClient, pr.cluster.ID, updatedNamespace.Name, metav1.ListOptions{
+		FieldSelector: "metadata.name=" + deployment.Name,
+	})
+	require.NoError(pr.T(), err)
+}
+
+func (pr *ProjectsTestSuite) TestMoveNamespaceOutOfProject() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a standard user and the user to the downstream cluster as cluster owner.")
+	standardUser, err := users.CreateUserWithRole(pr.client, users.UserConfig(), projects.StandardUser)
+	require.NoError(pr.T(), err, "Failed to create standard user")
+	standardUserClient, err := pr.client.AsUser(standardUser)
+	require.NoError(pr.T(), err)
+	err = users.AddClusterRoleToUser(pr.client, pr.cluster, standardUser, clusterOwner, nil)
+	require.NoError(pr.T(), err, "Failed to add the user as a cluster owner to the downstream cluster")
+
+	log.Info("Create a project in the downstream cluster and a namespace in the project.")
+	namespacePodLimit := "2"
+	projectPodLimit := "3"
```

Comment:
since we are not verifying anything wrt these, why do we want to update the limits here?

Reply:
good catch! removed it